demos/c_api_minimal_app/README.md

```diff
@@ -99,3 +99,65 @@ And then execute the alternative make target:
 cd demos/c_api_minimal_app
 make all_docker
 ```
+
+# Capi Benchmark
+
+This guide shows how to perform benchmark tests using OVMS's CAPI. 
```

Comment:
```suggestion
This guide shows how to perform benchmark tests using OpenVINO Model Server C-API.
It starts the model server and initiates a load from multithreaded clients sending sequential requests with synthetic data.
It can demonstrate expected throughput and latency for a specific model, configuration and C-API calls.

```